docs/identify.rst

```diff
@@ -0,0 +1,43 @@
+==============================
+Identifying Spectrum1D Formats
+==============================
+
+``specutils`` provides a convenience function,
+`~specutils.io.registers.identify_spectrum_format`, which attempts to guess the
+`~specutils.Spectrum1D` file format from the list of registered formats, and
+essentially acts as a wrapper on `~astropy.io.registry.identify_format`.
+
+This function is useful for identifying a spectrum file format without the need to
+read it in with Spectrum1d's `~specutils.Spectrum1D.read` method.  It uses the
+same identification method as ``read`` however, so it simply provides a convenience
+of access outside of calling ``read`` without any change in underlying functionality.
+It returns the best guess as to a valid format from the list of ``Formats``
+as given by `~astropy.io.registry.get_formats`.
+
+General usage is as follows, passing in an absolute filename path:
+
+.. code-block:: python
+
+    >>> from specutils.io.registers import identify_spectrum_format
+    >>> identify_spectrum_format("/path/to/file.fits")  # doctest: +SKIP
+
+An example is given to identify a SDSS MaNGA data cube file:
+
```

Comment:
I think the "general usage" isn't needed given that the MaNGA example is already quite clear
```suggestion
For eample, to identify a SDSS MaNGA data cube file:

```